packages/heartwood-components/components/09-sprucebot-avatar/sprucebot-typed-instructions.scss

```diff
@@ -23,11 +23,15 @@
 
 	&.large {
 		font-size: 2.5rem;
+
+		.cursor {
```

Comment:
Hmm. A bit subjective, but perhaps putting these props under `.cursor` props (i.e. `.cursor { .large & {} }`) would be clearer; otherwise it's a bit like, "what is cursor, does it have relative positioning?" when encountering this